### PR TITLE
various new tests

### LINKED ISSUE #9: highway = bus_stop en way
Les arrêts de bus sont censés être des nodes.
On peut lever une erreur sur les way avec le tag highway = bus_stop.

### PR DESCRIPTION
**Pull request by nlehuby**
* handle walking_bus as a public transport mode
* check if bus_stop is a node (Fix #9 )
* check if the line color is in a `colour` tag, 
* if the line color begins by `#`, check for the right format

### CODE REVIEW

File: transport.validator.mapcss [hunk 1]
```diff
@@ -149,6 +149,19 @@ relation.pt_route[tag(route)!=parent_tag(route_master)]
...
+    throwError: tr("Check the colour tag");
+}
+
+relation.pt_route[!colour][color],
+relation.pt_route_master[!colour][color]
+{
+    throwError: tr("The color of the public transport line should be in a colour tag");
+    fixChangeKey: "color=>colour";
+}
+
```
Comment (frodrigo) [inline transport.validator.mapcss]: There is already a more detailed mapcss check for `colour`. May be we can use and promote this one in place:
https://github.com/osm-fr/osmose-backend/blob/master/plugins/Colour.validator.mapcss

Comment (nlehuby) [reply transport.validator.mapcss]: Fun fact that the existing Mapcss file for colour codes does not check the actual `colour` tag! but sure, I can add it in this file.

But I'm considering to add it to the transport mapcss anyway:
- the colour mapcss file is not easy to add to JOSM. Most transport contributors won't add it at all
- a specific item/class in Osmose (or a tag, or some other way to filter them in the API) would be cool so this check can be considered in transport specific QA applications (such as Bifidus or custom scripts used by Jungle Bus on their missions). I want to be able to get these colour errors on pt lines, but I don't want to be pulluted by the colour erros on buildings. 

What do you think ? How bad would it be to have these errors duplicated in Osmose ?

Comment (nlehuby) [reply transport.validator.mapcss]: @frodrigo : would `-osmoseItemClassLevel: none;` do the trick to have the test in mapcss but not in Osmose (as it already exists in another form) ?

Comment (frodrigo) [reply transport.validator.mapcss]: Yes `osmoseItemClassLevel: none;` rules are filtered out.

Comment (nlehuby) [reply transport.validator.mapcss]: Thanks!

**Pull request merged**